src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -714,7 +714,7 @@ public void addChild(DLNAResource child, boolean isNew) {
 								// folder if supported/enabled and if it doesn't already exist
 								VirtualFolder transcodeFolder = getTranscodeFolder(true);
 								if (transcodeFolder != null) {
-									VirtualFolder fileTranscodeFolder = new FileTranscodeVirtualFolder(child.getDisplayName(), null);
+									VirtualFolder fileTranscodeFolder = new FileTranscodeVirtualFolder(child.getName(), null);
```

Comment:
@valib Why?

Reply:
Because it fixed the bug when TranscodeVirtualFolders name  was showed like e.g. ` [FFmpeg] {Audio: AC3/Engl...` instead of the media file name. I will check if your changes fixed it and possibly update the code.

Reply:
@valib My code already fixed it I think.

Reply:
I retest it and think that this code is better. Your changes works and fixed the bug but the TranscodeVirtualFolder names show not only the media file name but also subtitles which is not relevant in this case.